next/components/meta.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { NextSeo } from "next-seo";
+import * as React from "react";
+
+type Metatag = {
+  tag: string;
+  attributes: {
+    name?: string;
+    content?: string;
+    href?: string;
+    rel?: string;
+  };
+};
+
+type Metatags = Metatag[];
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  metatags?: Metatags;
+}
+
+export function Meta({ title, metatags }: MetaProps) {
+  const router = useRouter();
+
+  const { t } = useTranslation();
+
+  const getTag = React.useCallback(
+    (str: string, key: keyof Metatag["attributes"] = "name") => {
+      const result = metatags?.find((tag) => tag.attributes[key] === str);
+      return result?.attributes;
+    },
+    [metatags]
+  );
+
+  // We want to determine if we need to add the language path
+  // to create the canonical link for this page:
+  const languagePathFragment =
+    router.locale === router.defaultLocale ? "" : `/${router.locale}`;
+
+  const data = {
+    title: getTag("title")?.content ?? title,
+    description: getTag("description")?.content ?? t("meta-site-description"),
+    canonical: `${process.env.NEXT_PUBLIC_FRONTEND_URL}${languagePathFragment}${
+      router.asPath !== "/" ? router.asPath : ""
+    }`,
+    imageSrc: getTag("image_src", "rel")?.href,
+  };
+
+  data.title = data.title.concat(` | ${t("meta-site-name")}`);
+
+  return (
+    <NextSeo
+      title={data.title}
+      description={data.description}
+      canonical={data.canonical}
+      openGraph={{
+        title: title,
+        description: data.description,
+        type: "website",
+        url: data.canonical,
+        images: [
+          {
+            url: data.imageSrc ?? "",
+            alt: title,
+          },
+        ],
+      }}
+      additionalMetaTags={[
+        {
+          httpEquiv: "x-ua-compatible",
```

Comment:
these seemed generic enough that we might want to keep them? to show how to add other metatags?

Reply:
I agree, it's good to show as an example. But let's maybe drop the first "ie=edge" one (apparently it's not been necessary for while) and keep the second one? I don't have strong opinions about it though.

Reply:
ok! removed the first one in 3231192c4bba8b7ce055260915da1d965695c8f9